UPGRADE.md

```diff
@@ -1,8 +1,12 @@
-# Upgrading from v2.0.1 to v2.1.0
+# Upgrading from v2.1 to v2.2
+
+* The signature of `StorageInterface::resolveStream` method was changed. Nullable value of parameters `$fieldName`. 
```

Comment:
`$fieldName` is a single parameter, right?
Anyway, this sentence is not clear, please reword it as follows: "The `$fieldName` parameter is now nullable."

Reply:
Fixed. Sorry for my en.